### PR TITLE
chore: rename uibc quota params

### PR DESCRIPTION
**Pull request by robert-zaremba**
## Description

* Improve Quota docs
* rename uibc fields in genesis and params

## Summary by CodeRabbit

- **New Features**
  - Introduced a new mechanism for tracking inflows and outflows of tokens with resettable metrics and defined quotas.

- **Improvements**
  - Upgraded to Cosmos SDK v0.47 for enhanced performance and stability.
  - Adjusted governance parameters to refine the initial deposit ratio for proposals.

- **Refactor**
  - Renamed fields related to inflow and outflow sums for clarity and consistency.
  - Updated quota management logic to align with the new naming conventions.

- **Documentation**
  - Updated the README to reflect changes in the tracking and management of token inflows and outflows.

- **Bug Fixes**
  - Corrected the handling of zero-value quota parameters to prevent unintended behavior.

### CODE REVIEW

File: x/uibc/README.md [hunk 1]
```diff
@@ -25,28 +25,37 @@ IBC Quota is an upper limit in USD amount.
...
+#### Outflow Quota.
 
-- `Params.TokenQuota`: upper limit of a sum of all outflows per token. It's set to 1.2M USD per token. It limits the outflows value for each token.
-  NOTE: we measure per token as defined in the x/leverage, not the IBC Denom Path (there can be multiple paths). Since creating a channel is permission less, we want to use same quota token.
-- `Params.TotalQuota`: upper limit of a sum of all token outflows combined. For example if it's set to 1.6M USD then IBC outflows reaching the total quota will be 600k USD worth of ATOM, 500k USD worth of STATOM, 250k USD worth of UMEE and 250k USD worth JUNO.
+Inflows and outflows metrics above are used to **limit ICS-20 transfers** of tokens in the x/leverage Token Registry. The outflow transfer of token `X` is possible when:
 
-If a quota parameter is set to zero then we consider it as unlimited.
+1. Outflow quota after the transfer is not suppressed:
+1. `outflow_sum <= params.total_quota`. For example if it's set to 1.6M USD then IBC outflows reaching the total quota will be 600k USD worth of ATOM, 500k USD worth of STATOM, 250k USD worth of UMEE and 250k USD worth JUNO.
```
Comment (gsk967) [inline x/uibc/README.md:48]:
```suggestion
2. `outflow_sum <= params.total_quota`. For example if it's set to 1.6M USD then IBC outflows reaching the total quota will be 600k USD worth of ATOM, 500k USD worth of STATOM, 250k USD worth of UMEE and 250k USD worth JUNO.
```

File: x/uibc/README.md [hunk 2]
```diff
@@ -25,28 +25,37 @@ IBC Quota is an upper limit in USD amount.
...
+1. `token_quota[X] <= params.token_quota` - the token X quota is not suppressed.
```
Comment (gsk967) [inline x/uibc/README.md:49]:
```suggestion
3. `token_quota[X] <= params.token_quota` - the token X quota is not suppressed.
```

File: x/uibc/README.md [hunk 3]
```diff
@@ -25,28 +25,37 @@ IBC Quota is an upper limit in USD amount.
...
+1. OR Outflow quota lifted by inflows is not reached:
```
Comment (gsk967) [inline x/uibc/README.md:50]:
```suggestion
4. OR Outflow quota lifted by inflows is not reached:
```

File: x/uibc/README.md [hunk 4]
```diff
@@ -25,28 +25,37 @@ IBC Quota is an upper limit in USD amount.
...
+1. `outflow_sum <= params.inflow_outflow_quota_base + params.inflow_outflow_quota_rate * inflow_sum`
```
Comment (gsk967) [inline x/uibc/README.md:51]:
```suggestion
5. `outflow_sum <= params.inflow_outflow_quota_base + params.inflow_outflow_quota_rate * inflow_sum`
```

File: x/uibc/README.md [hunk 5]
```diff
@@ -25,28 +25,37 @@ IBC Quota is an upper limit in USD amount.
...
+1. `token_quota[X] <= params.inflow_outflow_token_quota_base + params.inflow_outflow_token_quota_rate * inflows[X]`
```
Comment (gsk967) [inline x/uibc/README.md:52]:
```suggestion
6. `token_quota[X] <= params.inflow_outflow_token_quota_base + params.inflow_outflow_token_quota_rate * inflows[X]`
```

File: x/uibc/quota/keeper/genesis.go [hunk 6]
```diff
@@ -33,11 +33,11 @@ func (kb Builder) ExportGenesis(ctx sdk.Context) *uibc.GenesisState {
...
 	return &uibc.GenesisState{
-		Params:          k.GetParams(),
-		Outflows:        outflows,
-		TotalOutflowSum: k.GetTotalOutflow(),
-		QuotaExpires:    *quotaExpires,
-		Inflows:         inflows,
-		TotalInflowSum:  k.GetTotalInflow(),
+		Params:       k.GetParams(),
+		Outflows:     outflows,
+		OutflowSum:   k.GetTotalOutflow(),
```
Comment (gsk967) [inline x/uibc/quota/keeper/genesis.go:38]:
```suggestion
		OutflowSum:   k.GetOutflowSum(),
```

**Review by gsk967 — APPROVED** [review]
Pre Appoving , I added small comments

**Review by toteki — APPROVED** [review]

**Pull request merged**